Mathlib/GroupTheory/Perm/Cycle/Concrete.lean

```diff
@@ -469,9 +467,32 @@ set_option linter.unusedTactic false in
 notation3 (prettyPrint := false) "c["(l", "* => foldr (h t => List.cons h t) List.nil)"]" =>
   Cycle.formPerm (Cycle.ofList l) (Iff.mpr Cycle.nodup_coe_iff (by decide))
 
-unsafe instance repr_perm [Repr α] : Repr (Perm α) :=
-  ⟨fun f _ => repr (Multiset.pmap (fun (g : Perm α) (hg : g.IsCycle) => isoCycle ⟨g, hg⟩)
-    (Perm.cycleFactorsFinset f).val -- toCycle is faster?
-    fun _ hg => (mem_cycleFactorsFinset_iff.mp (Finset.mem_def.mpr hg)).left)⟩
+/-- Represents a permutation as product of disjoint cycles:
+```
+#eval (1 : Perm (Fin 4))
+-- 1
+
+#eval (c[0, 1] : Perm (Fin 4))
+-- c[0, 1]
+
+#eval (c[0, 1] * c[2, 3] : Perm (Fin 4))
+-- c[0, 1] * c[2, 3]
```

Comment:
These three examples feel redundant. What about a series of examples like:
* Something which is already in disjoint cycles form, with the cycles in order
* Something which is already in disjoint cycles form, with the cycles not in order
* Something which is not in disjoint cycles form and computes to something other than 1
* Something which is not in disjoint cycles form and computes to 1

eg maybe `c[0, 1, 2, 3]`, `c[3, 1] * c[0, 2]`, `c[0, 1, 2] * c[1, 2, 3]`, `c[0, 1, 2] * c[1, 2, 3] * c[3, 1] * c[0, 2]`